### ISSUE TITLE
How to set behaviours

### ISSUE THREAD

**Issue by leshiple** [gribnoysup/react-yandex-maps#6]
Hi.
This example doesn't work.
```
<YMaps.Map
    width={width}
    height={height}
    state={{
        center: center,
        zoom: zoom,
    }}
    options={{
        searchControlProvider: 'yandex#search',
    }}
    behaviors={{
        disable:['drag', 'rightMouseButtonMagnifier']
    }}
>
```

**Comment by gribnoysup** [comment]
Hi @leshiple 

Behaviors are part of map state, according to [this doc](https://tech.yandex.com/maps/doc/jsapi/2.1/ref/reference/Map-docpage/#param-state.behaviors), so you can try doing this:

```js
<YMaps.Map
    width={width}
    height={height}
    state={{
        center: center,
        zoom: zoom,
        behaviors: [/* enabled behaviours */]
    }}
    options={{
        searchControlProvider: 'yandex#search',
    }}
>
```

The only downside will be that currently `state.behaviors` prop wouldn't be updated if it is changed. I'll try to fix this in next release. 

If you need to dynamically change map behavior, you can get an instance of map with `instanceRef` prop and change behavior with ymaps api methods (this solution is kinds hacky though)

```js
// ...
instanceRef={(map) => this.mapInstance = map}

// somewhere in the code of your component
this.mapInstance.behaviors.disable('drag')
this.mapInstance.behaviors.disable('rightMouseButtonMagnifier')
// ...
```

**Comment by leshiple** [comment]
Hi @gribnoysup. Thanks. It's cool!

**Comment by gribnoysup** [comment]
@leshiple I released v1.0.0, now behaviours are updated on props change
